### PR TITLE
Simple knife repath, removes a layer of unecessary nesting

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Ports:
- https://github.com/tgstation/tgstation/pull/62035

## About The Pull Request

Repaths all knives from `obj/item/kitchen/knife` to `obj/item/knife`, genericized knife behavior is now stored on the parent knife, which shouldnt be spawned in and only be used in checks. Children knives are written like `obj/item/knife/rainbow` or `obj/item/knife/combat`

There should be no ingame changes, other than I replaced a single Soviet kit to use a combat knife instead of a kitchen knife.

## Why It's Good For The Game

Kitchen knives were the first implemented knives in the game many years ago, and so thus all knives derive from it and use its path as a parent. This works fine, but it can be somewhat confusing in coding or trying to debug why heretic blades or rainbow knives are considered a 'kitchen' tool.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

All typical knife behavior

https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/647a4af9-22b1-46de-89d5-881b99e8306c

</details>

## Changelog
:cl: rkz, itseasytosee
code: repathed knives, removing unecessary nesting
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Update the UpdatePaths/scripts to include the repathed items so that in the case of merge conflicts, we can just run the script and resolve them

**Comment by PowerfulBacon** [comment]
image

File: tools/UpdatePaths/Scripts/9994_knives.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/item/knife/ : /obj/item/kitchen/knife/{@OLD}
```
Comment (PowerfulBacon) [inline tools/UpdatePaths/Scripts/9994_knives.txt:1]: Should this have the / at the end?

Comment (Tsar-Salat) [reply tools/UpdatePaths/Scripts/9994_knives.txt:1]: Yes. Those are every knife besides kitchen knives.

/obj/item/kitchen/knife is a kitchen knife. Before it was the parent of all knives, but now it is a derivative of /obj/item/knife. This differentiation is important so regular kitchen knives are converted

In the unlikely event a merge conflict occurs, I will fix it within the hour.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**